src/background/responders.ts

```diff
@@ -44,12 +56,20 @@ export const responders: { [T in Action['type']]: Responder<T> } = {
   UPDATE_EDITOR_STATE(state, action): Partial<AppState> {
     const tab = ensureActiveTab(state)
 
+    const handle = tab.feedbackState.hostTwitterHandle.handle
+    const { editorState } = action.payload
+    const status = editorState.getCurrentContent().getPlainText('\u0001')
+
+    // If the new editor state doesn't start with the handle, don't update the store
+    if (handle && !status.startsWith(`${handle} `)) return {}
```

Comment:
I don't understand why this is happening

Reply:
This is what makes the handle not editable if it exists

Reply:
Got it! Let's update the comment to say that. Do you think we should include an alert as well?

Reply:
I think it's clear as is, but I'm not against adding an alert. What would the alert be? "You can't edit the handle"? I think users won't like this alert